src/blog/v8-release-89.md

```diff
@@ -0,0 +1,42 @@
+---
+title: 'V8 release v8.9'
+author: 'Ingvar Stepanyan, awaiting a call'
+avatars:
+ - 'ingvar-stepanyan'
+date: 2021-02-03
+tags:
+ - release
+description: 'V8 release v8.9 brings performance improvements to calls with argument size mismatch.'
+tweet: ''
+---
+Every six weeks, we create a new branch of V8 as part of our [release process](https://v8.dev/docs/release-process). Each version is branched from V8’s Git master immediately before a Chrome Beta milestone. Today we’re pleased to announce our newest branch, [V8 version 8.9](https://chromium.googlesource.com/v8/v8.git/+log/branch-heads/8.9), which is in beta until its release in coordination with Chrome 89 Stable in several weeks. V8 v8.9 is filled with all sorts of developer-facing goodies. This post provides a preview of some of the highlights in anticipation of the release.
+
+## JavaScript
+
+### Top-level await
+
+[Top-level await](https://github.com/tc39/proposal-top-level-await) is available in the [Blink rendering engine](https://www.chromium.org/blink) 89, a primary embedder of V8.
```

Comment:
```suggestion
[Top-level `await`](https://v8.dev/features/top-level-await) is available in the [Blink rendering engine](https://www.chromium.org/blink) 89, a primary embedder of V8.
```

Reply:
Best to consistently point to our feature explainer IMHO. People can find the proposal link from there if they want more info.

Reply:
I thought about this too, but wasn't sure because it kind of renders the following sentence useless? (its only purpose is to provide a link to our explainer)

Reply:
Should I remove the follow-up sentence?

Reply:
Up to you :) I think it's fine either way